tests/network/primary_pod_network.go

```diff
@@ -98,12 +98,12 @@ var _ = SIGDescribe("Primary Pod Network", func() {
 
 				It("should report PodIP/s IPv4 as its own on interface status", func() {
 					vmiPod := tests.GetRunningPodByVirtualMachineInstance(vmi, vmi.Namespace)
-					Eventually(vmiIP).Should(Equal(vmiPod.Status.PodIP), "should contain VMI Status IP as Pod status ip")
-					Eventually(vmiIPs).Should(ContainElement(vmiPod.Status.PodIP), "should contain IPv4 reported by guest agent")
+					Eventually(vmiIP, 2*time.Minute, 5*time.Second).Should(Equal(vmiPod.Status.PodIP), "should contain VMI Status IP as Pod status ip")
+					Eventually(vmiIPs, 2*time.Minute, 5*time.Second).Should(ContainElement(vmiPod.Status.PodIP), "should contain IPv4 reported by guest agent")
 				})
 
 				It("should report VMIs static IPv6 at interface status", func() {
-					Eventually(vmiIPs).Should(ContainElement(libnet.DefaultIPv6Address), "should contain IPv6 address set by cloud-init and reported by guest agent")
+					Eventually(vmiIPs, 30*time.Second, 2*time.Minute).Should(ContainElement(libnet.DefaultIPv6Address), "should contain IPv6 address set by cloud-init and reported by guest agent")
```

Comment:
this one as well please ;]

Reply:
good catch! [fixed](https://github.com/kubevirt/kubevirt/compare/581b94f2e799ece09b037230717dc2521140f8c0..93d88cc57e26acf688c27c0b96bcbc1e690856c6)